lib/vrf.c

```diff
@@ -1064,6 +1064,7 @@ static int lib_vrf_create(struct nb_cb_create_args *args)
 
 	vrfp = vrf_get(VRF_UNKNOWN, vrfname);
 
+	SET_FLAG(vrfp->status, VRF_CONFIGURED);
```

Comment:
vrf_set_user_cfged() and unset appropriate places.

Reply:
Fixed.